frame.go

```diff
@@ -66,12 +67,16 @@ func getStringForType(typ MessageType) string {
 
 func parseFrame(m string, typ MessageType, hof headerOrFooterMarker) (brand string, err error) {
 
+	// strip whitespace
+	re := regexp.MustCompile("[>\\s]+")
+	s := strings.TrimSpace(re.ReplaceAllString(m, " "))
+
 	sffx := getStringForType(typ)
 	if len(sffx) == 0 {
 		err = makeErrBadFrame("Message type %v not found", typ)
 		return
 	}
-	v := strings.Split(m, " ")
+	v := strings.Split(s, " ")
```

Comment:
We do, that was what caused travis to fail earlier. I've since fixed it with a `strings.TrimSpace` call on line 72.